contract/p/gnoswap/rbac/ownable.gno

```diff
@@ -31,7 +31,7 @@ func (o *Ownable2Step) TransferOwnershipBy(newOwner, caller address) error {
 		return ErrUnauthorized
 	}
 
-	if newOwner.String() == "" || !newOwner.IsValid() {
+	if newOwner == "" || !newOwner.IsValid() {
```

Comment:
Instead of directly comparing against an empty string, comparing directly against the `zeroAddress` variable would make the intent clearer.